interp/constrextern.ml

```diff
@@ -69,7 +69,7 @@ let print_no_symbol = ref false
 (* Turning notations and scopes on and off for printing *)
 module IRuleSet = Set.Make(struct
     type t = interp_rule
-    let compare x y = Pervasives.compare x y
+    let compare x y = compare x y
```

Comment:
Why not use the Util.pervasives_compare here? (and a few other places)

Reply:
I didn't bother as there is no consistency on the codebase, only used in places were another `compare` was in scope.